SpanishFilter/sections/specific.txt

```diff
@@ -19,6 +19,8 @@ atribuna.com.br,jovemnerd.com.br,laopiniondemurcia.es,larazon.es,dicio.com.br,ba
 brasilescola.uol.com.br##.publi-dir
 brasilescola.uol.com.br##.corpoPubli
 brasilescola.uol.com.br##.area-publi-topo
+economia.ig.com.br##div[class^="maislidas"]
+economia.ig.com.br###containerComp
```

Comment:
It seems this is not related to ads. I see these elements at https://ultimosegundo.ig.com.br/policia/2022-05-06/ataque-faca-alunos-feridos-escola-ilha-governador-rj.html and useful content may be blocked.